### DISCUSSION TITLE
Add an "Hide beatmaps already downloaded" option in beatmap listing

### DISCUSSION THREAD

**Discussion by Acumena** [ppy/osu#20363] [Ideas] [answered]
![Screenshot(1)](https://user-images.githubusercontent.com/108238851/190900941-05e7ec9a-6f29-4465-b625-9bbe7bf6d33d.png)

This would help people find beatmaps to download without having to worry about downloading ones they already have.

**Comment by Theighlin** [comment, marked as answer]
Duplicate of #15727

**Reply by Acumena** [reply]
Sorry didn't know.
